service-automation/src/com/appyschool/suiteSetup/TestSuiteSetup.java

```diff
@@ -409,40 +411,68 @@ public void testCreateCoCurricularAreas(HashMap<String, String> map) throws JSON
 		verifyEquals(responseObj.getString("status"), "OK");
 	}
 	
-	
 	/**
-	 * Create a Students
+	 * Create a Student
 	 * @param map
 	 * @throws JSONException
 	 */
-	@Test(dataProviderClass = com.appyschool.common.CommonUtils.class, dataProvider = "getDataFromFile", dependsOnMethods="testMapBranchStandardSection")
+	
+	@Test(dataProviderClass = com.appyschool.common.CommonUtils.class, dataProvider = "getDataFromFile",dependsOnMethods="testMapBranchStandardSection")
 	@DataProviderArguments("./testData/configuration/student.csv")
 	public void testCreateStudents(HashMap<String, String> map) throws JSONException {
 		n++;
 		String jsonTemplate = ConfigProperties.getValue(GenericConstants.STUDENT_CREATE_PAYLOAD);
 		
-		Map<String, String> branchStandardIds = CommonMethods.getBranchStandardFromMapBranchStandardSections(SuiteSetupProperties.getValue("BRANCH_ID_1"));
+		Map<String, String> branchStandardIds = CommonMethods.getBranchStandardFromMapBranchStandardSections(SuiteSetupProperties.getValue("BRANCH_ID_"+n));
 		
-		map.put("$branchId", SuiteSetupProperties.getValue("BRANCH_ID_1"));
+		map.put("$branchId", SuiteSetupProperties.getValue("BRANCH_ID_"+n));
 		map.put("$branchStandardSection", branchStandardIds.get("mapId"));
 		map.put("$standard", branchStandardIds.get("standardId"));
-
 		
 		JSONObject json = CommonUtils.getJsonFromTemplate(jsonTemplate, map);
+
 		json.put("idNumber", "Appy" + RandomStringUtils.randomNumeric(5));
 		json.put("rollNumber", RandomStringUtils.randomNumeric(5));
-		json.put("primaryPhoneNumber", RandomStringUtils.randomNumeric(5) + RandomStringUtils.randomNumeric(5));
-		json.put("alternatePhoneNumber", RandomStringUtils.randomNumeric(5) + RandomStringUtils.randomNumeric(5));
+		json.put("primaryPhoneNumber", RandomStringUtils.randomNumeric(10));
+		json.put("alternatePhoneNumber", RandomStringUtils.randomNumeric(10));
 		json.put("primaryEmailId", map.get("$firstName") + RandomStringUtils.randomNumeric(2) + "@appyschool.com");
 		json.put("secondaryEmailId", map.get("$firstName") + RandomStringUtils.randomNumeric(2) + "@appyschool.com");
 		Response response = StudentServices.createStudent(json.toString());
 		response.then().log().all();
 		
 		JSONObject responseObj = new JSONObject(response.getBody().asString());
-		verifyEquals(response.statusCode(), 200);
-		verifyTrue(responseObj.get("id") != null);
-		verifyEquals(responseObj.getString("status"), "OK");
+		SoftAssertionBase.verifyEquals(response.statusCode(), 200);
+		SoftAssertionBase.verifyEquals(responseObj.getString("status"), "OK");
```

Comment:
move "OK" as constant in some Constants file, and use that variable for assertions.